WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationLocalStore.swift

```diff
@@ -59,25 +58,29 @@ public protocol ConversationLocalStoreProtocol {
     /// - Parameter isFederationEnabled: A flag indicating whether a `Federation` is enabled.
 
     func storeConversation(
-        _ conversation: WireAPI.Conversation,
+        _ conversation: WireDomain.Conversation,
```

Comment:
https://github.com/wireapp/wire-ios/pull/2098/commits/94b4ad4ad0103ea30b3f2355baaee2a074dd5797 introduces a WireDomain conversation object that replaces the WireAPI.Conversation object to avoid introducing API layer elements in our local stores. (local stores should be agnostic of the API layer).